packages/reports/addon/components/missing-intervals-warning.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ *   {{missing-intervals-warning
+ *       response=response
+ *       onDetailsToggle=(action 'resizeVisualization')
+ *   }}
+ */
+import Component from '@ember/component';
+import { computed, get, getWithDefault } from '@ember/object';
+import Moment from 'moment';
+import layout from '../templates/components/missing-intervals-warning';
+
+const DATE_FORMAT = 'YYYY[/]MM[/]DD';
+
+export default Component.extend({
+  layout,
+
+  /**
+   * @property {String} tagName
+   */
+  tagName: '',
+
+  /**
+   * @property {Array} classNames
+   */
+  classNames: ['missing-intervals-warning'],
+
+  /**
+   * @property {Array} missingIntervals - The formatted intervals displayed in the details section
+   */
+  missingIntervals: computed('response', function() {
+    let rawIntervals = get(this, 'response.meta.missingIntervals') || [];
+
+    return rawIntervals.map(interval => {
+      //Make both dates in the interval inclusive
+      let dates = interval.split('/'),
+          start = new Moment(dates[0]).format(DATE_FORMAT),
+          end   = new Moment(dates[1]).subtract(1, 'second').format(DATE_FORMAT);
+
+      //If the interval only covers one day, return just that date
+      if(start === end) {
+        return start;
+      }
+
+      return [start, end].join(' - ');
+    });
+  }),
+
+  /**
+   * @property {Boolean} showDetails - Determines whether the component is expanded or collapsed
+   */
+  showDetails: false,
+
+  /**
+   * @property {Boolean} warningEnabled - Shows the warning if missing intervals are present
+   */
+  warningEnabled: computed.notEmpty('missingIntervals'),
+
+  actions: {
+    toggleDetails() {
+      let onDetailsToggle = getWithDefault(this, 'onDetailsToggle', () => {});
+      this.toggleProperty('showDetails');
+      setTimeout(onDetailsToggle, 300);
```

Comment:
Can we schedule for later in the route action instead of the `setTimeout`?
https://www.emberjs.com/api/ember/3.1/functions/@ember%2Frunloop/later